lightning-background-processor/src/lib.rs

```diff
@@ -234,7 +254,13 @@ impl BackgroundProcessor {
 
 				if updates_available {
 					log_trace!(logger, "Persisting ChannelManager...");
-					persister.persist_manager(&*channel_manager)?;
+					channel_manager_persister.persist_manager(&*channel_manager)?;
+					if let Some(ref handler) = net_graph_msg_handler {
+						if network_graph_persister.persist_graph(handler.network_graph()).is_err() {
```

Comment:
We definitely don't need to persist the graph here, otherwise we'll be doing it every time we have any HTLC/commitment updates.

Reply:
that's actually related to the question I had in the PR description... I added it here in order to [test if NetworkGraph persists](https://github.com/lightningdevkit/rust-lightning/blob/cc8853362274612e4858c7ffc792789934671167/lightning-background-processor/src/lib.rs#L604-L611) without waiting for 60 seconds for the [prune block](https://github.com/lightningdevkit/rust-lightning/blob/cc8853362274612e4858c7ffc792789934671167/lightning-background-processor/src/lib.rs#L302-L312) to run. 

I was wondering if you had any ideas on that

Reply:
You're welcome to const-ify that 60 and then use `#[cfg(test)]` to have a different value in test, eg the way `PING_TIMER` and `FRESHNESS_TIMER` are done.

Reply:
@TheBlueMatt ah, got it! didn't see that. Thanks! 😄